### ISSUE TITLE
ver 0.8.1.30 - interaction problems

### ISSUE THREAD

**Issue by knord1** [mezz/JEIAddons#4]
using jei 16.2.78

No crash, but problems interacting with the world.

There seems to something stopping Thaumcraft 5.1.3 from making infusion (tried adding Grinning Devil Faceplate to Thaumium fortress helm). The wand just bobs like an interaction should happen, but nothing does.

**Comment by mezz** [comment]
I have a hard time imagining how JEIAddons would cause this problem...
Please reload your game and see if it still doesn't work.
Then, remove JEIAddons and see if it works.

JEIAddons is client-side-only, and the server decides if the infusion works or not, so I don't think there's any way for JEIAddons to break this.

**Comment by knord1** [comment]
That was what I thought too.

I did remove all mods I had and only kept Froge, JEI and Thaumcraft. It worked fine then.
Then I added back the content mods. Interaction was fine.
Then I added jeiaddon and infusion just wont work. I have no clue why this addon should affect this but with all my mods added, jeiaddon removed, the interaction works.

Or it could be forge have changed something? :/

modlist:
BetterFps-1.2.0.jar
BiomesOPlenty-1.8.9-3.0.0.1632-universal.jar
BloodMagic-1.8.9-2.0.0-4.jar
buildcraft-7.2.0-pre6.jar
Chameleon-1.8.8-0.2.0.jar
ComputerCraft1.77pr0.jar
craftingtweaks-mc1.8.8-3.0.2.jar
FastLeafDecay-1.8-1.6.jar
InGameInfoXML-1.8.9-2.8.1.80-universal.jar
inventorysorter-1.8.9-0.7.1+23.jar
InventoryTweaks-1.60-dev-28.jar
ironchest-1.8.9-6.0.115.759.jar
jei_1.8.9-2.16.2.78.jar
journeymap-1.8.8-5.1.3-unlimited.jar
JustEnoughResources-1.8.9-0.0.3 (1).jar
LLOverlayReloaded-1.0.4-mc1.8.8.jar
LunatriusCore-1.8.9-1.1.2.32-universal.jar
Mantle-1.8.8-0.6.jar
mcjtylib-1.8.9-1.8.1beta4.jar
MmmMmmMmmMmm-1.9b.jar
RandomThings-MC1.8.8-3.5.7.jar
rftools-1.8.9-4.20beta8.jar
StorageDrawers-1.8.8-2.2.0.jar
TConstruct-1.8.8-2.0.1.jar
Thaumcraft-1.8.9-5.1.3.jar
Waila-1.6.0-B3_1.8.8.jar
VersionChecker-1.2.5.jar
Baubles-1.8.9-1.1.3.0.jar

Forge: 11.15.0.1697

**Comment by mezz** [comment]
I will try to reproduce the issue.
What wand are you using? Can you post a picture of the infusion setup?

**Comment by knord1** [comment]
![2016-01-14_03 33 22](https://cloud.githubusercontent.com/assets/8095995/12314251/99c8299a-ba6f-11e5-99cb-a0f5ec48fdb8.png)
Brass Topped Silverwood Wand

I tried it in a new empty world too (creative mode, Thaumium Capped Silverwood Wand), same results.

**Comment by mezz** [comment]
I don't think the Thaumium fortress helm will work if it's cheated in.

**Comment by knord1** [comment]
It works, the fortress helmet doesn't have any (known to me) extra data.
And I tried cheating in the helmet (in creative world). With jeiaddons, infusion didn't work, without jeiaddons infusion worked.

In my "real" world, I created the helmet some days ago (three maybe?) and that required an infusion that worked.
But I have updated most files since then
And today I tried the infusion for the evil grinn, but it just didn't work.
After a few hours adding/removing mods jeiaddons was the result.
With jeiaddons in my "real" world, infusion don't work.
Witout jeiaddons in my "real" world, infusion works.

I have no log errors, nothing that indicated that something is wrong.

(Thank you for taking time with this!)

**Comment by mezz** [comment]
Just bizarre. I stepped through in debug mode but didn't find anything that depended on the client. I wonder if it's only able to cause issues on integrated servers.
I need to work on something else right now but I will come back to this later.

**Comment by knord1** [comment]
An new test.

With only Thaumcraft, Jei and JeiAddons (same versions as above, jei 2.17.3.83).
Using a Thaumium capped Silverwood Wand.

Tried to make the infusion altar "Mystical Contruct" without any knowledge, nothing happens (that's ok) and the wand gives a bob animation.
Learned the creative Thaumanomicon and then I could create the altar.

Made the Thaumium Fortress Helm with infusion, worked fine.

Tried to made the Grinning Devil Faceplate, did not work.

It acts like I don't know the receipt (like when I tried to make the infusion altar without knowledge).
